### PR TITLE
Update dropdowns styling

### PR DESCRIPTION
**Pull request by inikolova**
Related to https://github.com/telerik/kendo-angular-dropdowns/issues/104

### CODE REVIEW

File: scss/common/_forms.scss [hunk 1]
```diff
@@ -58,6 +58,7 @@
         border-style: solid;
         box-sizing: border-box;
         position: relative;
+        transition: all .1s ease; // sass-lint:disable-block no-transition-all
```
Comment (gyoshev) [inline scss/common/_forms.scss:61]: Do we really need to transition all properties? It seems like `transition: color .1s ease, background .1s ease;` will be sufficient.

File: scss/popup/_theme.scss [hunk 2]
```diff
@@ -29,7 +29,10 @@ $popup-text: $normal-text-color !default;
 
     .k-list {}
 
-    .k-list .k-item {}
+    .k-list .k-item {
+        transition: all .2s ease; // sass-lint:disable-block no-transition-all
```
Comment (gyoshev) [inline scss/popup/_theme.scss:33]: Ditto.

**Comment by gyoshev** [comment]
Other than the minor transition question, the commits look great! :+1:

File: scss/common/_forms.scss (hunk 1 above)
Comment (yordanov) [reply scss/common/_forms.scss:61]: `border-color` and `box-shadow` should also be subject to transition, as they change through different states too.

Comment (gyoshev) [reply scss/common/_forms.scss:61]: I see. Seems like `all` is okay for brevity.

**Pull request merged**